src/data/ModuleData.js

```diff
@@ -129,6 +129,7 @@ markdown: `
     title: "JS Comments",
     markdown: `
 # Learning Module 4
+Hangman Module
```

Comment:
is this supposed to be here? looks like this change was made in the middle of a different data object?

Reply:
No, you're right that shouldn't be there. I'm guessing I was planning to replace an earlier module and realized it was a much better idea just to add one to the end and forgot to delete that.

Reply:
Just fixed that.